map.json

```diff
@@ -43,5 +43,19 @@
 		"id" : "win-enable-nested-hyperv",
 		"path" : "src/windows/win-enable-nested-hyperv.ps1",
 		"description" : "Enables nested hyperV using attached data disk as OS disk."
+	},
+	{
+		"id" : "win-dumpconfigurator",
```

Comment:
Why is there two win-dumpconfigurator? I think these are duplicates?

Reply:
Also the safe mode code shouldn't be added to this PR. I think something went wrong during the merge from master when you were resolving conflicts?